manage/sawtooth_manage/vnm.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+from multiprocessing import Process
```

Comment:
You could do this with threading, which would probably be desirable.  The primary use case for multiprocessing is offloading CPU.